akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -272,11 +272,11 @@ final case class RootActorPath(address: Address, name: String = "/") extends Act
 
   override val toSerializationFormat: String = toString
 
-  override def toStringWithAddress(addr: Address): String =
+  override def toStringWithAddress(newAddress: Address): String =
```

Comment:
Ping @ktoso or @2m is this perhaps source incompatible but MiMa only cares about binary incompatibilities?